docs/content/middlewares/http/overview.md

```diff
@@ -145,6 +145,7 @@ http:
 | [IPWhiteList](ipwhitelist.md)             | Limits the allowed client IPs                     | Security, Request lifecycle |
 | [InFlightReq](inflightreq.md)             | Limits the number of simultaneous connections     | Security, Request lifecycle |
 | [PassTLSClientCert](passtlsclientcert.md) | Adds Client Certificates in a Header              | Security                    |
+| [Plugin](../../../plugins/index.md)       | Use community plugin                              | All                         |
```

Comment:
`Plugin` is not a middleware but a way to configure community plugins. So, we think that this line should be removed.

```suggestion
```

Reply:
How about adding a link from middleware to plugins ?